ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -2355,6 +2356,62 @@ def test_tensorflow_unravel_index(
     )
 
 
+# sequence_mask
+@st.composite
+def _sequence_mask_helper(draw):
+    max_val = draw(st.integers(min_value=1, max_value=100000))
+    in_dtype, lens = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("valid"),
+            num_arrays=1,
+            min_value=1,
+            max_value=max_val,
+        )
+    )
+
+    max_len = draw(st.integers(min_value=int(ivy.max(lens)), max_value=max_val))
+    dtype = draw(st.sampled_from([
+    'float16',
+    'uint8',
+    'complex128',
+    'bool',
+    'float64',
+    'int8',
+    'int16',
+    'complex64',
+    'float32',
+    'int32',
+    'int64']))
```

Comment:
Why don't you use `helpers.get_dtypes("valid")`?
```suggestion
    dtype = draw(helpers.get_dtypes("valid"))
```

Reply:
1) This draws a list of dtypes instead of a single dtype.
2) This ignores a few dtypes that I believe should be considered by the testing function

Reply:
Makes sense, as it only allow you to draw from supported dtypes. Which are `lens` dtypes.